testapi.pm

```diff
@@ -1384,7 +1384,7 @@ sub send_key_until_needlematch {
         wait_screen_change {
             send_key $key;
         };
-        if (!$counter--) {
+        if (!--$counter) {
```

Comment:
```suggestion
        if (--$counter <= 0) {
```
So we get into this condition even if somebody passes 0.

Reply:
I guess you mean `>= 0`  ?

Reply:
Good idea! I'd also propose a check if passed parameter '$counter' is >0 ; passing 0 or negative number does not make any sense in my opinion.  What do you think ?

Reply:
It should be good enough now.